guides/manual-installation.md

```diff
@@ -479,54 +468,43 @@ e859c07888ca   registry.suse.com/trento/trento-wanda:1.2.0   "/bin/sh -c '/app/b
 
 Both containers should be running and listening on the specified ports.
 
-### Setup nginx as reverse proxy
-
-#### Option 1: Creating a Self-Signed Certificate
-
-> Note: This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the OpenSSL documentation. Remember to replace example.com with your domain.
+### Validate the health status of trento web and wanda
+Validate the health status of the Trento web and wanda services locally by accessing the ```healthz``` and ```readyz``` api.
 
-**Step 1**: Generate a private key:
+Test Trento web health status with curl:
 
-```bash
-openssl genrsa -des3 -out trento.key 2048
+```
+curl http://localhost:4000/api/readyz; curl http://localhost:4000/api/healthz; echo
 ```
 
-**Step 2**: Generate a CSR (Certificate Signing Request):
-
-```bash
-openssl req -new -key trento.key -out trento.csr
+Test Trento wanda health status with curl:
+```
+curl http://localhost:4001/api/readyz; curl http://localhost:4001/api/healthz; echo
 ```
 
-Follow the prompts to enter your information.
+Expected output if Trento web/wanda is ready and the database connection is setup correctly:
+```
+{"ready":true}{"database":"pass"}
+```
 
-**Step 3**: Remove passphrase from the key
+### Prepare SSL certificate and install Nginx
 
-```bash
-cp trento.key trento.key.org
-openssl rsa -in trento.key.org -out trento.key
-```
+Create or provide a certificate for [nginx](https://nginx.org/en/) to enable SSL for Trento.
 
-**Step 4**: Create config file for SAN:
+#### Option 1: Creating a Self-Signed Certificate
 
-> Note: See [X509 V3 certificate extension configuration format documentation](https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html) for more details.
+> **Note:** This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the [OpenSSL documentation]((https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html)).
```

Comment:
Now that I think about it, it might be wise to add a note here so that people that go with the self-signed certificate option know that they have to follow the steps described in `#### Configuring the Agent host with the Self-Signed Certificate`.